tests/integration/components/settings/contact-info-test.js

```diff
@@ -0,0 +1,10 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('settings/contact-info-section', 'Integration | Component | settings/contact info');
```

Comment:
Change `Integration | Component | settings/contact info ` to `Integration | Component | settings/contact info section` to match the component name